log/levels/levels.go

```diff
@@ -0,0 +1,127 @@
+package levels
+
+import "github.com/go-kit/kit/log"
+
+// Levels provides a leveled logging wrapper around a logger. It has five
+// levels: debug, info, warning (warn), error, and critical (crit). If you
```

Comment:
@sasha-s Since this PR has already been merged I recommend that you write up the features you want as separate issues and we'll iterate on them.